core/modules/replica/VerifyJob.h

```diff
@@ -248,23 +238,44 @@ class VerifyJob
       */
     void notify(util::Lock const& lock) final;
 
+private:
+
+    /**
+     * Construct the job with the pointer to the services provider.
+     *
+     * @see VerifyJob::create()
+     */
+    VerifyJob(size_t maxReplicas,
+              bool computeCheckSum,
+              CallbackTypeOnDiff const& onReplicaDifference,
+              Controller::Ptr const& controller,
+              std::string const& parentJobId,
+              CallbackType const& onFinish,
+              Job::Options const& options);
+
     /**
      * The callback function to be invoked on a completion of each request.
      *
-     * @param request - a pointer to a request
+     * @param request
+     *   a pointer to a request
      */
-    void onRequestFinish(FindRequest::Ptr request);
+    void _onRequestFinish(FindRequest::Ptr const& request);
 
     /**
      * Find the next replicas to be inspected.
      *
-     * @param lock        - the lock must be acquired by a caller of the method
-     * @param replicas    - a collection of replicas returned from the database
-     * @param numReplicas - a desired number of replicas to be pulled from the database
+     * @param lock
+     *   the lock must be acquired by a caller of the method
```

Comment:
For all of these lock comments, the really important thing is the name of the underlying mutex. It's probably worth putting the name of the mutex in the variable name or something as passing a Lock with the wrong mutex could make debugging more difficult.

Reply:
Good suggestion! Thank you!